lib/ddtrace/configuration/settings.rb

```diff
@@ -361,6 +372,7 @@ def initialize(*_)
       # @see https://docs.datadoghq.com/agent/troubleshooting/site/
       # @default `DD_SITE` environment variable, otherwise `nil` which sends data to `app.datadoghq.com`
       # @return [String,nil]
+      # @configure_with Datadog
       option :site do |o|
```

Comment:
This will most likely be used by tracing customers in 1.x if/when it goes agentless. If it's only a matter of changing the comment/description in the mean time, that's fine.

Reply:
Yeah, I meant the comment, since right now it's only a "nothing to see here, move along" :)